wt-status.h

```diff
@@ -66,6 +82,8 @@ struct wt_status {
 	int show_branch;
 	int hints;
 
+	struct wt_status_vvp vvp;
```

Comment:
Having thought about this more, maybe we do not even *want* to introduce a new `struct` here, but just a

```c
unsigned char sha_commit[GIT_SHA1_RAWSZ]; /* Commit SHA (when not Initial) */
```

and skip the `enabled` field in favor of `verbose`?